app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -44,7 +45,12 @@ class MemberPaymentsTransformer @Inject()(
       schemeMadeTransferOut = memberPayments.memberDetails.exists(_.transfersOut.nonEmpty),
       lumpSumReceived = memberPayments.lumpSumReceived,
       pensionReceived = false,
-      surrenderMade = false,
+      surrenderMade = memberPayments.benefitsSurrenderedDetails match {
+        case SectionDetails(made @ true, completed @ true) => true
+        case SectionDetails(made @ true, completed @ false) => false
```

Comment:
I'm not sure this behaves as we expected in ETMP. We need to verify this via QA.

@harishmuddegowda @BabuRupoll7623